### ISSUE TITLE
Open multiple select input until user clicks away

### ISSUE THREAD

**Issue by trishtzy** [react-materialize/react-materialize#658]
### Description

Input multiple select closes after an option is selected. It should stay open until user clicks away from the option dropdown.

See multiple select section in https://materializecss.com/select.html

### Steps to Reproduce

1. Use 
```
<Input multiple={true}>
    <option value='1'>Option 1</option>
    <option value='2'>Option 2</option>
    <option value='3'>Option 3</option>
</Input>
```
2. Select 'Option 1'

**Expected behavior:** [What you expect to happen]
Dropdown option stays open

**Actual behavior:** [What actually happens]
Dropdown option closes

### Versions

**react-materialize:** 2.1.2
**materialize-css:** 0.100.2
**react:** 16.3.2

**Comment by trishtzy** [comment]
Opened a PR [here](https://github.com/react-materialize/react-materialize/pull/660)

**Comment by trishtzy** [comment]
PR approved. Closing issue.

**Comment by alextrastero** [comment]
If you add `Fixes #658` in the description of your PR it will close this issue automatically once merged :)

**Comment by trishtzy** [comment]
oh cool ok! thanks!
